app/models/repository.rb

```diff
@@ -6,7 +6,7 @@ class Repository < ApplicationRecord
   before_validation :truncate_description
 
   belongs_to :language, optional: true
-  has_many :pull_requests, dependent: :destroy
+  has_many :issues, dependent: :destroy
```

Comment:
The `pull_requests` relationship did not appear meaningful, from what I could tell, as we are counting PRs completely separately. There is a column to count PRs on repos, but that is something else altogether.

To test that this wouldn't disrupt our user flow, I set the competition to be active, and destroyed all repos (and associated issues). My winning user's PRs were still there and winning, so I think that answers that question.

Reply:
Yup, I never used repo when I created the PR model.